ngraph/test/type_prop/logical_and.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "util/type_prop.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+TEST(type_prop, logical_and_incorrect_type_f32)
+{
+    auto input1 = make_shared<op::Parameter>(element::f32, Shape{1, 3, 6});
+    auto input2 = make_shared<op::Parameter>(element::f32, Shape{1, 3, 6});
+    try
+    {
+        auto logical_and = make_shared<op::v1::LogicalAnd>(input1, input2);
+    }
+    catch (const NodeValidationFailure& error)
+    {
+        EXPECT_HAS_SUBSTRING(error.what(),
+                             std::string("Operands for logical operators must have boolean element type but have element type f32"));
+    }
+
```

Comment:
Consider close body of this test to function. IMHO this will be more readable and allows easily find difference between this test